squad/frontend/group_settings.py

```diff
@@ -76,6 +78,65 @@ def form_valid(self, form):
         return redirect(reverse('home'))
 
 
+class GroupRelationshipForm(forms.ModelForm):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields['group'].widget = forms.HiddenInput()
+        self.fields['group'].disabled = True
+
+
+class GroupMemberForm(GroupRelationshipForm):
```

Comment:
It would be nice to have alphabetic ordering on users field. Right now it's most likely using pk as ordering field.